google/cloud/networkconnectivity/quickstart/WORKSPACE.bazel

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A minimal WORKSPACE file showing how to use the Network Connectivity API
+# C++ client library in Bazel-based projects.
+workspace(name = "networkconnectivity_quickstart")
+
+# Add the necessary Starlark functions to fetch google-cloud-cpp.
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# Fetch the Google Cloud C++ libraries.
+# NOTE: Update this version and SHA256 as needed.
+http_archive(
+    name = "com_github_googleapis_google_cloud_cpp",
+    sha256 = "a7e51bfffb95a377094b2ae7e3b9f715a68ed931c48992c7273b2fae989c029c",
+    strip_prefix = "google-cloud-cpp-1.36.0",
+    url = "https://github.com/googleapis/google-cloud-cpp/archive/v1.36.0.tar.gz",
```

Comment:
My observation is that renovate bot will not update this until the next release.  Can you update this to point to v2.2.0?

Reply:
Done.... although keeping the version up to date will be a problem for future libraries that are GA upon generation.